tests/FakeItEasy.Analyzer.Tests/UnusedReturnValueAnalyzerTests.cs

```diff
@@ -110,6 +110,39 @@ interface IFoo { int Bar(); }
         [Test]
         [SetUICulture("en-US")] // so that the message is in the expected language regardless of the OS language
         [SuppressMessage("Microsoft.Naming", "CA2204:Literals should be spelled correctly", MessageId = "CallTo", Justification = "It's an identifier")]
+        public void Diagnostic_Should_Be_Triggered_When_Call_Specification_Is_Not_Used_With_UsingStatic()

```

Comment:
Something a little more like

```c#
Diagnostic_Should_Be_Triggered_When_Call_Specification_Made_In_Global_Scope_Is_Not_Used
```

I've read docs that say that the "using static" "[renders static methods available in global scope](https://msdn.microsoft.com/en-us/magazine/dn879355.aspx)", which is why I put "global scope" in there. And I think deduplicating "use" in the name helps a little.

I'm not sure the method name is quite right yet, but I _think_ it's a step in the right directin.